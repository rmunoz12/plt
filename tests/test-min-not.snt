@setup { }

@turns {

func void begin() {
    print(5, "\n", not True, "\n", -3, "\n");
    end();
}

}
