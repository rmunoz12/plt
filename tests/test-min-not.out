5
false
-3